mobx_codegen/pubspec.yaml

```diff
@@ -14,7 +14,8 @@ environment:
 dependencies:
   path: ^1.6.2
   source_gen: ^0.9.4
-  mobx: ^0.3.3
+  mobx:
+    path: ../mobx
```

Comment:
This should point to the pub-dev package (`^x.y.z`) as before and not local. Else publishing would not work. You can use the [dependency_overrides](https://dart.dev/tools/pub/dependencies#dependency-overrides) option instead.